README.md

```diff
@@ -62,7 +62,7 @@ Code should be committed as follows:
     https://github.com/matrix-org/matrix-react-sdk/tree/master/res/themes
 
 React components in matrix-react-sdk come in two different flavours:
-'structures' and 'views'. Structures are stateful components which handle the
+`'structures'` and `'views'`. Structures are stateful components which handle the
```

Comment:
In my opinion we should only use backticks if it is code (e.g. shell commands, CSS classes etc.). Here it isn't. The quotes here indicate that these are specific terms. Can you remove the backticks here?

```suggestion
'structures' and 'views'. Structures are stateful components which handle the
```